spec/index.html

```diff
@@ -410,13 +410,15 @@ <h3>Quoted Literals</h3>
 
```

Comment:
Note, I changed back the previous language for "may not" to "must not", as this is an informative section. The Erratum was directed to the normative media type registration, I believe.